archive.js

```diff
@@ -114,9 +114,16 @@ const uploadForm = (req, res) => {
 exports.uploadForm = uploadForm;
 
 const uploadProcess = (req, res) => {
-	var litematic = req.files.litematic;
-	litematic.mv(`./archive/${litematic.name}`);
+    const file = req.files.file;
+    const file_ext = (file.name.split('.').pop())
+    console.log(file_ext)
 
-	res.redirect('/archive');
+    if (file_ext !== "litematic" && file_ext !== "schematic" && file_ext !== "nbt" && file_ext !== "png") {
+        res.redirect('/')
```

Comment:
we should send a message explaining why you arent permitted to upload that file or something
```suggestion
        res.send('Please only upload .litematic, .schematic, .nbt or .png files.');
```